### PR TITLE
Add new study_session type

### PR DESCRIPTION
**Pull request by kalletlak**
Fix https://github.com/cBioPortal/cbioportal/issues/7104

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -127,6 +127,7 @@ curl --user user:pass
 | Type  | Description  |
 |---|---|
 | main_session | represent result's page query |
+| study_session | represent study page state |
```
Comment (inodb) [inline README.md]: @kalletlak what exactly does this represent. Is it the URL filter? If so maybe good to be explicit. Also how it differs from virtual_study. I guess for one you store the URL filter, for the other you store the list of sample ids. I.e. one is static the other is dynamic.

As a side note: an alternative approach might be to extend the virtual_study in some way, cause we are already storing the filters in there if i remember correctly

Comment (kalletlak) [reply README.md]: Yes, `study_session` just stores filter state. But `virtual_study` has both filter and list of sample ids.

True, we could combine them. We should discuss about it.

cc: @alisman

**Comment by inodb** [comment]
@kalletlak since this was changed: https://github.com/cBioPortal/cbioportal-frontend/pull/3038. I guess we can close this PR?

**Comment by inodb** [comment]
gonna close for now but feel free to re-open

**Pull request closed without merging**